examples/core/instancing/app.js

```diff
@@ -47,21 +51,13 @@ const animationLoop = new AnimationLoop({
     });
 
     const pickInfo = pickModels(gl, {models: [cube], position: pickPosition, framebuffer});
-    // console.log(pickInfo ? pickInfo.color : 'picked background');
     cube.updateModuleSettings({
-      pickingSelectedColor: pickInfo && pickInfo.color
+      pickingSelectedColor: pickInfo && pickInfo.color,
+      pickingValid: pickInfo !== null
     });
 
     gl.clear(GL.COLOR_BUFFER_BIT | GL.DEPTH_BUFFER_BIT);
     cube.render();
-  },
-  onMouseMove({gl, tick, cube, framebuffer}) {
```

Comment:
It would be neat to integrate with luma's `addEvents` here. Or even align with the new event handling in deck.gl. 

Will require some discussion, maybe open an issue...